scripts/upgrade

```diff
@@ -87,44 +87,11 @@ then
 
     # Remove the option backup_core_only after the backup.
     ynh_app_setting_delete $app backup_core_only
-
-    ynh_clean_setup () {
```

Comment:
Uuuh, in fact, why did you removed the ynh_clean_setup function?

Reply:
true, it's a mistake